packages/mermaid/src/diagrams/flowchart/flowDb.ts

```diff
@@ -507,9 +491,9 @@ export const addSubGraph = function (_id, list, _title) {
     return { nodeList, dir };
   }
 
-  let nodeList = [];
+  let nodeList: string[] = [];
 
-  const { nodeList: nl, dir } = uniq(nodeList.concat.apply(nodeList, list));
+  const { nodeList: nl, dir } = uniq(list.flat());
   nodeList = nl;
```

Comment:
If we're not using `nl`, we can just directly set `nodeList` from `uniq(list.flat());`

```suggestion
  const { nodeList, dir } = uniq(list.flat());
```